### DISCUSSION TITLE
Unable to delete file api.log - locked by gauge

### DISCUSSION THREAD

**Discussion by antondebeer** [getgauge/gauge#2491] [General]
Hi,
Can someone please give advice how to "unlock" the txt file api.log from gauge. I can't do my gradle clean task due to this issue. I get the error: What went wrong:
Execution failed for task ':clean'.
> java.io.IOException: Unable to delete file '...\logs\api.log'

**Comment by chadlwilson** [comment]
Sounds like windows - perhaps need to look for rogue java processes that are still running/stuck and kill them?

If it's a gradle daemon/worker process you interrupted might need to `gradle --stop` or `gradlew --stop`?

**Reply by antondebeer** [reply]
Yes sorry it is a windows machine. i did and i get No Gradle daemons are running.

**Reply by chadlwilson** [reply]
Look for other java processes perhaps, or use a windows tool like sysinternals pprocess explorer to confirm which specific process has the file locked? (You can text search for open file handles in such tools)

If it's the gauge daemon and it's running because an IDE needs it, you might have to exclude from your gradle clean. I'm not familiar with the specific log off top of my head so these are just generic suggestions.
